servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingToStreamingService.java

```diff
@@ -87,8 +88,7 @@ protected void handleSubscribe(final Subscriber<? super StreamingHttpResponse> s
                             // https://tools.ietf.org/html/rfc7230#section-3.3.2
                             HttpHeaders headers = metaData.headers();
                             boolean addTrailers = metaData.version().major() > 1 || isTransferEncodingChunked(headers);
-                            if (!addTrailers && metaData.version().major() == 1 && metaData.version().minor() > 0 &&
-                                    !hasContentLength(headers) &&
+                            if (!addTrailers && metaData.version() == HTTP_1_1 && !hasContentLength(headers) &&
```

Comment:
I expected the opposite change :)
It seems more reliable to check for `metaData.version().major() == 1 && metaData.version().minor() > 0` in case HTTP/1.2 will ever happen. All minor versions have to be backward compatible with previous versions.

Reply:
I think `== HTTP_1_1` is fine in practice, but I'll switch back to the other conditional as it is more general. I'll also move this into a common utility method so it will be easier to detect where this clause is checked in the future.